lib/bundlex/toolchain/xcode.ex

```diff
@@ -14,15 +14,8 @@ defmodule Bundlex.Toolchain.XCode do
     sources_part = nif.sources |> Enum.map(fn source -> "\"#{source}\"" end) |> Enum.join(" ")
     libs_part = nif.libs |> Enum.map(fn lib -> "-l#{lib}" end) |> Enum.join(" ")
 
-    pkg_configs_part =
-      nif.pkg_configs
-      |> Enum.map(fn pkg_config ->
-        %Porcelain.Result{status: 0, out: out} =
-          Porcelain.exec("pkg-config", ["--cflags", "--libs", pkg_config])
-
-        out |> String.trim()
-      end)
-      |> Enum.join(" ")
+    {pkg_configs_part, 0} = System.cmd("pkg-config", ["--cflags", "--libs" | nif.pkg_configs])
```

Comment:
When `nif.pkg_configs` is an empty list, this command returns `1`, what results in match error. Please add a check for this, and maybe separate all the stuff to a helper function, e.g. in the `Bundlex.Toolchain` module.